.github/workflows/builds.yml

```diff
@@ -654,6 +657,7 @@ jobs:
           CFLAGS: "-fprofile-instr-generate -fcoverage-mapping -O0"
       - name: Extracting suricata-verify
         run: tar xf prep/suricata-verify.tar.gz
+      - run: rm -r prep
```

Comment:
Does the github action to upload the coverage report only need the generated coverage.txt?

I think I've seen this mostly failing as that action is download some files it needs before upload, but we've already generated the coverage.txt.  Maybe we could delete all the profiling files after the coverage.txt is created. That should free up a lot of diskpace before the coverage action does its thing.

Reply:
yeah but the issue is that we first generate almost 10GiB worth of profraw files, then merge them and generate the report. I think to really address this we'd need to incrementally merge those files, so after each test or group of tests, or find another way to not have to store that much intermediate state.

Reply:
Ok, I had not seen it fail during generation yet.  The download before upload is 165M, which isn't much I guess.

Reply:
We should use a rustc wrapper to avoid computing the coverage of external crates...

Reply:
Trying something in https://github.com/OISF/suricata/pull/13025 because adding brotli led to Fedora 41 (Suricata Verify codecov) failure